qt_gui_cpp/src/qt_gui_cpp_shiboken/CMakeLists.txt

```diff
@@ -49,13 +51,15 @@ if(shiboken_helper_FOUND)
     set(qt_gui_cpp_BINDINGS "${qt_gui_cpp_BINDINGS}" PARENT_SCOPE)
 
     set(QT_INCLUDE_DIR "${Qt5Widgets_INCLUDE_DIRS}")
-    shiboken_generator(libqt_gui_cpp global.h typesystem.xml ${PROJECT_SOURCE_DIR}/src/qt_gui_cpp_shiboken "${qt_gui_cpp_shiboken_SRCS}" "${qt_gui_cpp_HDRS}" "${qt_gui_cpp_INCLUDE_PATH}" "${CMAKE_CURRENT_BINARY_DIR}")
+    string(REPLACE ";" ":" qt_gui_cpp_INCLUDE_PATH_WITH_COLONS "${qt_gui_cpp_INCLUDE_PATH}")
```

Comment:
Necessary since the argument `INCLUDE_PATH` of the `shiboken_generator()` function doesn't accept lists atm. (This could be patched in `python_qt_binding` if we want to do that instead but I wanted to keep the number of PRs minimal to get this passing first.)

Reply:
Adding a comment before this command would be good for posterity.

Reply:
I would rather do follow up PRs to make the function accept a proper list.

Reply:
If https://github.com/ros-visualization/python_qt_binding#92 gets merge before this PR I can update the patch to not work around the problem here.

Reply:
Removed the workaround in 8804a738b6403b3104865a8892c597c132adbda6.